encoding/mvt/prepare.go

```diff
@@ -4,14 +4,18 @@ import (
 	"log"
 
 	"github.com/go-spatial/geom"
-	"github.com/go-spatial/tegola"
 )
 
-// PrepareGeo converts the geometry's coordinates to tile coordinates
-func PrepareGeo(geo tegola.Geometry, tile *tegola.Tile) geom.Geometry {
+// A default valu for pixelExtent for use with PrepareGeo
+const DefaultPixelExtent = 4096.0
```

Comment:
Why not use the mvt.DefaultExtent?

Reply:
I used pixel extent in this package because I think extent would be confusing considering the use of `geom.Extent` within this package. That's also why I left the the parameter named `tile`

Reply:
I'm referring to the package var that already exists: https://github.com/go-spatial/geom/blob/master/encoding/mvt/mvt.go#L9

Reply:
I forgot that exists, I'll change that